docs/core/base/document.md

```diff
@@ -0,0 +1,112 @@
+# Document
+
+The term "Document" is used in the only in the code. It refers a single piece of content that created by Elementor, such as a post, page, or a saved template.
+
+Each document has its own type and features.
+
+A document is saved as a `post` in the WordPress `wp_posts` table. His elements and settings are saved as metadata in the `wp_postmeta` table.
```

Comment:
This whole introduction doesn't really feel smooth to read. It feels like a bunch of list items. Maybe try to make it more fluent